src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -97,5 +111,11 @@ export const navPaths = [
     path: '/workspaces/:namespace/:name/applications/:application',
     component: ApplicationLauncher,
     title: ({ name, application }) => `${name} - ${application}`
+  },
+  {
+    name: 'workspace-spark-interface-launch',
+    path: '/workspaces/:namespace/:name/applications/:application/:sparkInterface',
+    component: ApplicationLauncher,
+    title: ({ name, application, sparkInterface }) => `${name} - ${application} - ${sparkInterface}`
   }
```

Comment:
Added this path here in AppLauncher to leverage the tab launching logic existed there for Terminal and RStudio.
I wanted to roll the new path into the path above it via

```javascript
name: 'workspace-application-launch',
path: '/workspaces/:namespace/:name/applications/:application/:sparkInterface',
component: ApplicationLauncher,
title: ({ name, application }) => `${name} - ${application} - ${sparkInterface}`
```

But it doesn't seem to work for the cases where `application` is `Terminal` or `RStudio`, in which case there is no `sparkInterface`  🤷🏼

Reply:
The title is a simple string displayed at the top as I understand it. Should it be something like this:

`title: ({ name, application }) => `${name} - ${application}${sparkInterface ? ' -' + sparkInterface + ' ': ''}`

Reply:
Did you mean for the case where we'd use `path: '/workspaces/:namespace/:name/applications/:application/:sparkInterface'` for both Spark UIs and applications? If so, the issue is, `sparkInterface` is expected to be input at all times, which doesn't work for non-Spark UI applications.